hack/verify-govet.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+GO_VERSION=($(go version))
+
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4') ]]; then
+  echo "Unknown go version '${GO_VERSION}', skipping go vet."
+  exit 0
+fi
+
+MODE=${1:-quiet}
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
+
+find_files() {
+  find . -not \( \
+      \( \
+        -wholename './output' \
+        -o -wholename './_output' \
+        -o -wholename './.git' \
+        -o -wholename './release' \
+        -o -wholename './pkg/assets/bindata.go' \
+        -o -wholename './pkg/assets/*/bindata.go' \
+        -o -wholename './target' \
+        -o -wholename '*/third_party/*' \
+        -o -wholename '*/Godeps/*' \
+      \) -prune \
+    \) -name '*.go'
+}
+
+files=$(find_files)
+
+tests="printf methods structtags composites asmdecl assign atomic bool buildtags copylocks nilfunc rangeloops unreachable shadow unsafeptr unusedresult"
+whitelist="assign atomic bool copylocks nilfunc rangeloops shadow unreachable unsafeptr"
+
+mkdir -p _tmp/govet
+
+for test in $tests
+do
+  if [ "$MODE" == "verbose" ] || [ $(echo "${whitelist}" | grep -o "${test}") ]
+  then  
+    touch "_tmp/govet/${test}.log"
```

Comment:
Still it would make sense to gather even those in `_output` or make sure to add `_tmp` to `.gitignore` otherwise I'm pretty sure somebody might commit that by mistake. Although keeping all of build/prof/etc. artifacts in one `_output` dir is more preferable imho.